onboarding and removing organizers (#148)
rladies/rladiesguide: content/organization/intro/co-organizers/index.en.md

If some people are happy to help without committing, it's already a great progress for you.
Remind them regularly they could become an official organizer. :wink:

## How to onboard new organizers?

> * The current organizers should contact the onboarding team at chapters@rladies.org, who will guide them through the process of onboarding new organizers and should share the email addresses of the new organizers.

Reviewer: ```suggestion
* The current organizers should contact the onboarding team at chapters@rladies.org. Please share the name and email address of each new organizer so that we can guide everyone through the onboarding process.
```

  
* Before a volunteer commits as an organizer, an idea might be to have them host (i.e. organize) one or a few meetups (be careful to acknowledge their work although they're not an official organizer (yet)!) --- see [Meetup docs about the "event organizer" role](https://help.meetup.com/hc/en-us/articles/360002879411-Managing-a-leadership-team).

## How to remove retired chapter organizers?

> * The current organizers should reach out to the onboarding team at chapters@rladies.org, who will guide them through the process of removing retired chapter organizers.

Reviewer: ```suggestion
* The current organizers should reach out to the onboarding team at chapters@rladies.org, who will guide them through the process retiring a former organizer.
```

## How to organize team work?

* You might assign roles to organizers e.g. one would be in charge of social media for instance;

> * Or task assignment can be more flexible. Organizers will probably volunteer to do things that play to their strenghts. You can have a rotation system for some tasks like Mastodon and Meetup management.

Reviewer: ```suggestion
* Or task assignment can be more flexible. Organizers will probably volunteer to do things that play to their strenghts. You can have a rotation system for some tasks like social media and Meetup management.
```

If some people are happy to help without committing, it's already a great progress for you.
Remind them regularly they could become an official organizer. :wink:

## How to onboard new organizers?

* Before a volunteer commits as an organizer, an idea might be to have them host (i.e. organize) one or a few meetups (be careful to acknowledge their work although they're not an official organizer (yet)!) --- see [Meetup docs about the "event organizer" role](https://help.meetup.com/hc/en-us/articles/360002879411-Managing-a-leadership-team);

* After that:

>   * Send new organizers the [R-Ladies Organizers form](https://rladies.org/form/organiser).

Author: Hello everyone,
We have the following private issue ( https://github.com/rladies/new-chapters-onboarding/blob/main/.github/ISSUE_TEMPLATE/existing-chapter-update.md) for tracking updates on existing chapters. @cosimameyer, I'm not sure if you have access to it. We could suggest that the current organizer either follow the steps mentioned to onboard new organizers or contact the chapters@rladies.org team. In that case, the chapter onboarding team can handle the process by opening the issue and tracking the necessary steps.
I don’t think it makes sense to notify the chapter onboarding team just to open the issue if the current organizers feel comfortable following the guide and completing the steps themselves. Or maybe it's up to the chapter onboarding team to open an issue and handle these steps regardless? In that case, this section should be presented differently, without requiring the current organizer to go through the onboarding steps for new organizers. What do you think?

@acolum please feel free to add your thoughts if needed!

>   * Send new organizers the [R-Ladies Organizers form](https://rladies.org/form/organiser).

Reviewer: The process that @acolum described in the issue template seems very well organized. We can ask the new organizer to update the information on GitHub themselves if they feel comfortable doing so or request help from someone on the team. That can be discussed when they contact us.
I'm not sure if contacting the onboarding team should be presented as optional. We need people to inform us about upcoming changes in their chapters so we can open an issue and track all the necessary steps. Currently, many chapters have added organizers on Meetup but haven't filled out the form, added the organizer to GitHub, etc. If they don’t use Slack, we have no way to contact them.
Maybe we could take a minimal approach here: just telling people that for onboarding and retiring organizers they need to contact us via email. That wil also make easier to maintain these parts of the Guide.  
In the [Coordination](https://guide.rladies.org/coordination/onboarding/) section we can describe the process of opening an issue and any other step we need to consider. In this part, which is intended for organizers, it might be better to keep things simple and straightforward

>   * Send new organizers the [R-Ladies Organizers form](https://rladies.org/form/organiser).

Author: Thank you so much for your feedback, @rivaquiroga. I understand what you mean and agree with your suggestion. I'll move the details I've added about onboarding and retiring organizers to the Coordination section, and keeping the process simple in the Organizers section by directing the organizers to contact the onboarding team to onboard or retire organizers.

>   * Invite new organizes to the R-Ladies Community Slack.

Author: Absolutely! Thank you.
